invenio_oauthclient/contrib/settings.py

```diff
@@ -46,11 +47,24 @@ def __init__(
             },
         }
 
+        signup_options = signup_options or {}
+        signup_options.setdefault("auto_confirm", True)
+        signup_options.setdefault("send_register_msg", False)
+
+        if (
+            not signup_options["auto_confirm"]
+            and not signup_options["send_register_msg"]
+        ):
+            raise ValueError(
```

Comment:
I am not 100% about this check... is it something that we should enforce? Without, it can create issues (user cannot confirm).

Reply:
In my opinion, if we don't have a way to resend the confirmation e-mail then it's better to have the check than not .
However it's possibly an underlying issue that an user can end up in that situation in the first place. It might even happen that, by some unexpected issue, the e-mail is not sent / received and the user is left in the "void".

Reply:
I think to full-proof this check, you would also need to check if "login without confirmation" is possible (maybe similar to [the first variable checks as done here](https://github.com/inveniosoftware/flask-security-fork/blob/3001b0785d9ee769dae52484657e37e2733505db/flask_security/confirmable.py#L61-L62)). That helps with the case where confirmation is required to login, and the configuration here would break it.

Reply:
In this case, the `CONFIRMABLE` will be true, so the `registration msg` must be sent right?
From the doc, `LOGIN_WITHOUT_CONFIRMATION` allows to login even before user's confirm, but the e-mail is sent.
```
Specifies if a user may login before confirming their email when the value of SECURITY_CONFIRMABLE is set to True
```

Honestly, I think we should remove this check completely, to avoid blocking any use case I cannot think ok... WDYT?

Reply:
Yeah, maybe it's best not to assume any of these (e.g. maybe someone wants to "manually confirm" users via backoffice), so let's remove :)